netmaster/master/network.go

```diff
@@ -95,6 +95,7 @@ func CreateNetwork(network intent.ConfigNetwork, stateDriver core.StateDriver, t
 	}
 
 	subnetIP, subnetLen, _ := netutils.ParseCIDR(network.SubnetCIDR)
+
```

Comment:
Do we really need this in this PR ? 
@shaleman Could you comment on whitespaces and new lines addition in a change. I guess gofmt should take care of this.